### DISCUSSION TITLE
Using network shares to plot

### DISCUSSION THREAD

**Discussion by kayjay010101** [swar/Swar-Chia-Plot-Manager#985] [Q&A]
For my use case I have a multi-flash server connected with 56Gb ethernet to my main computer. The idea is to plot using my main computer and have both plotting and dest. drives all on the server (plotting to the flash array, destionation is a direct-connected JBOD with HDDs shared over the network). So far this has worked great by simply setting these settings:
temporary_directory: \\SERVER_NAME\temp
temporary2_directory: 
destination_directory: \\SERVER_NAME\plots
However, this means that the plot manager cannot read the share's status and does not display % used and other very useful metrics.  Simply using drive letters means that the plot manager will not work at all, so I have to use the full UNC path. I've tried using symbolic links and that helps, but then it just shows the status of the drive the link is mounted on (in my case, a 5TB plotting share is linked to C:\chiatemp, which then shows the C: drive as the plotting drive, which is a measly 500GB and has 90% usage, none of which the plots take up... so the data is useless to me.).

Is there a way for the plot manager to plot on (specifically SMB) network shares and also display drive status?

**Comment by ethanthekiwi** [comment]
I don't know what OS you are running, but I wonder if using NFS instead of SMB might work better?

**Reply by kayjay010101** [reply]
Windows 10, I'll try NFS.
Problem is RDMA is only possible with SMB in Windows so I'll have crazy high CPU usage with NFS at 56Gb.
